pkg/diskmaker/diskmaker.go

```diff
@@ -310,3 +341,95 @@ func fileExists(filename string) bool {
 	}
 	return true
 }
+
+// getPVCreationLock checks whether a PV can be created based on this device
+// and Locks the device so that no PVs can be created on it while the lock is held.
+// the PV lock will fail if:
+// - another process holds an exclusive file lock on the device (using the syscall flock)
+// - a symlink to this device exists in symlinkDirs
+// returns:
```

Comment:
This list of returns doesn't seem to be correct any longer.

Reply:
fixed